app/models/company.rb

```diff
@@ -1,6 +1,6 @@
 class Company < ActiveRecord::Base
-  has_many :users
-  has_many :notes
+  has_many :users, dependent: :destroy
```

Comment:
Dependent has something to do with caching, right Orion?  Are we caching companies anywhere?  Or is this doing something else?